components/StoryCard.tsx

```diff
@@ -108,15 +112,14 @@ const StoryCard = ({ story, state, index, addFilter }: StoryCardParams): JSX.Ele
             {...provided.dragHandleProps}
             ref={provided.innerRef}>
             <CardContainer
-              width="250px"
               hoverable
               onClick={e => {
                 if (['A', 'SPAN', 'TEXTAREA'].includes(e.target.nodeName)) {
                   return;
                 }
                 dispatch(selectStory(story));
               }}>
-              <Card.Content>
+              <Card.Content style={contentStyle}>
```

Comment:
You may prefer to use styled-components here!

Reply:
Done, could not override the Content's default 16pt padding without `!important` tho